main.go

```diff
@@ -93,13 +96,49 @@ func (c *Config) init() error {
 		aws.NewConfig().WithRegion(c.awsKinesisRegion),
 	)
 
+	filters := os.Getenv("CT_EVENT_FILTERS")
+	if filters != "" {
+		c.eventFilters = parseFilters(filters)
+	}
+
 	return nil
 }
 
 type CloudTrailFile struct {
 	Records []map[string]interface{} `json:"Records"`
 }
 
+type EventFilter struct {
+	EventName   string
+	EventSource string
+}
+
+func parseFilters(filters string) []*EventFilter {
+	var eventFilters []*EventFilter
+	for _, filter := range strings.Split(filters, ",") {
+		event_filter := strings.Split(filter, ":")
+		eventFilters = append(eventFilters, newEventFilter(event_filter[0], event_filter[1]))
```

Comment:
this looks like it could result in a slice indexing panic if the environment variable isn't formatted properly

Reply:
:+1: Added check to skip incorrectly formatted filters.